packaging/flavorgen/flavors/generators.go

```diff
@@ -386,7 +385,7 @@ func kubeVIPPod() string {
 						},
 						{
 							Name:  "vip_interface",
-							Value: vipNetworkInterfaceVar,
+							Value: "eth0",
```

Comment:
Want to add a comment why this is hardcoded now?

Reply:
this shouldn't have been templated in the first place I think